indicators/smbc-acab82b5.yml

```diff
@@ -0,0 +1,23 @@
+title: SMBC Phishing Kit acab82b5
+description: |
+    Detects a SMBC phishing kit targeting Japanese users.
+
+references:
+  - https://urlscan.io/result/acab82b5-6182-4cab-96b1-7e2af19b668b
+  - https://urlscan.io/result/a8a41bab-97ed-43d8-85d8-d760161ab317
+  - https://urlscan.io/result/607f6acb-1301-4ca5-9e33-0e0ca5b7c359
+  - https://urlscan.io/result/6c29c34f-1dac-433c-b2d9-005bd8db3ee1
+
+
+detection:
+    FormContains: 
+        html|contains: 
+            - 'method="post" id="tijiao" action="1.php"'
+    SMBCTitle:
+        title: '三井住友カード会員向けサービス「Vpass」ログイン'
+
```

Comment:
I would include the high entropy string set as the iframe element id `aMpc0Wu2zFxeefIt` rather than the page title as it seems to be also used on the original website